frontend/src/metabase-types/store/app.ts

```diff
@@ -7,7 +9,13 @@ export interface AppErrorDescriptor {
   context?: string;
 }
 
+export interface AppBreadCrumbs {
```

Comment:
@npfitz You also need to add `createMock` functions for the mocked store state. They are in the `mocks` folder here